### PR TITLE
Create disease_series_by_translocation.yaml

### LINKED ISSUE #2550: create design pattern for cancer chromosomal locations
For terms like this one from Orphanet:
[Term]
id: MONDO:0035087
name: acute myeloid leukemia with t(9;22)(q34.1;q11.2)
subset: ordo_disease
is_a: MONDO:0000001 {source="Orphanet:585867"} ! disease or disorder
is_a: MONDO:0020078 {source="Orphanet:585867"} ! acute myeloid leukemia with recurrent genetic anomaly
xref: Orphanet:585867 {source="MONDO:equivalentTo"} ! acute myeloid leukemia with t(9;22)(q34.1;q11.2)

related to https://github.com/monarch-initiative/mondo/issues/2540

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #2550

### CODE REVIEW

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+pattern_name: disease_series_by_translocation
+
+description: >-
+  This pattern will be used for diseases that are caused by a specific chromosomal translocation. The content below is just a placeholder, as the class to be used in the variable is to be determined.
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:4]: remember the pattern for patterns. This is too general. It makes it sound like any subclass of chromosomal disease should have this. 

In fact it is *only* for disease *classes* that are of the form "{translocation} disease"

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+pattern_name: disease_series_by_translocation
+
+description: >-
+  This pattern will be used for diseases that are caused by a specific chromosomal translocation. The content below is just a placeholder, as the class to be used in the variable is to be determined. 
+  
+  Examples: [Orphanet:585867 acute myeloid leukemia with t(9;22)(q34.1;q11.2)](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=29204&Disease_Disease_Search_diseaseGroup=585867&Disease_Disease_Search_diseaseType=ORPHA&Disease(s)/group%20of%20diseases=Acute-myeloid-leukemia-with-t-9-22--q34-1-q11-2-&title=Acute%20myeloid%20leukemia%20with%20t922q34.1q11.2&search=Disease_Search_Simple), [NCIT:C68659 Childhood B Acute Lymphoblastic Leukemia with t(9;22)(q34.1;q11.2); BCR-ABL1](https://ncithesaurus.nci.nih.gov/ncitbrowser/pages/concept_details.jsf?dictionary=NCI_Thesaurus&version=20.11e&code=C68659&ns=ncit&type=properties&key=null&b=1&n=0&vse=null)
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:6]: use Mondo example

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+equivalentTo:
+    text: "%s and 'relation TBD' some %s
+    vars:
+      - disease
+      - translocation
+
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/disease_series_by_gene.yaml
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:46]: why is this not at the top?

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      
+def:
+    text: Any %s in which the cause of the disease is a translocation at %s.
+    vars:
+      - disease
+      - translocation
+
+
+equivalentTo:
+    text: "%s and 'relation TBD' some %s
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:40]: add an annotation to the pattern that says this is a proposed pattern, cc @matentzn

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    text: '%s with %s'
+    vars:
+      - disease
+      - translocation
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: '%s, %s'
+    vars:
+      - gene
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:29]: ??

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml (hunk 1 above)
Comment (cmungall) [reply src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:4]: oh I take it back. Actually we need two patterns

 - D caused by T
 - any disease caused by T

also is it only translocations?

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+pattern_name: disease_series_by_translocation
+
+description: >-
+  This pattern will be used for diseases that are caused by a specific chromosomal translocation. The content below is just a placeholder, as the class to be used in the variable is to be determined. 
+  
+  Examples: [Orphanet:585867 acute myeloid leukemia with t(9;22)(q34.1;q11.2)](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=29204&Disease_Disease_Search_diseaseGroup=585867&Disease_Disease_Search_diseaseType=ORPHA&Disease(s)/group%20of%20diseases=Acute-myeloid-leukemia-with-t-9-22--q34-1-q11-2-&title=Acute%20myeloid%20leukemia%20with%20t922q34.1q11.2&search=Disease_Search_Simple), [NCIT:C68659 Childhood B Acute Lymphoblastic Leukemia with t(9;22)(q34.1;q11.2); BCR-ABL1](https://ncithesaurus.nci.nih.gov/ncitbrowser/pages/concept_details.jsf?dictionary=NCI_Thesaurus&version=20.11e&code=C68659&ns=ncit&type=properties&key=null&b=1&n=0&vse=null)
+  
+classes:
+    disease: MONDO:0000001
+    translocation: TBD
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:10]: give an example. E..g https://civicdb.org/events/genes/4/summary/variants/1/

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+relations:
+    TBD: TBD
```
Comment (cmungall) [inline src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:13]: @kshefchek @deepakunni3 @mbrush what relation do we use to connect a disease to a variant?

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml (hunk 4 above)
Comment (matentzn) [reply src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:40]: Thats exactly why we are using a different directory in upheno. I can see you prefer a status tag :P

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml (hunk 3 above)
Comment (matentzn) [reply src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:46]: @nicolevasilevsky Send me a perfectly ordered pattern with IRI/name at the top and I will write a script to order all our patterns in the same way

File: src/patterns/dosdp-patterns/disease_series_by_translocation.yaml (hunk 7 above)
Comment (kshefchek) [reply src/patterns/dosdp-patterns/disease_series_by_translocation.yaml:13]: similar to what we would use for gene to disease, for clinvar we use the pathogenic and likely_pathogenic relations from GENO:0000840 and GENO:0000841

for this we could use RO:0003303 causes condition or is that too strong?

**Comment by nicolevasilevsky** [comment]
this is no longer needed

**Pull request closed without merging**